packages/vue/src/Form/Select/OcSelect.vue

```diff
@@ -245,6 +246,14 @@ const onUpdateDropdown = () => {
   }, 10)
 }
 
+const loadMore = (e) => {
+  const scrollDifference = e.target?.scrollHeight - Math.round(e.target?.scrollTop)
+
+  if (scrollDifference <= e.target?.clientHeight) {
+    emit('loadMore')
+  }
```

Comment:
when `loadMore` is emitted & main application is called api asynchronous, 
user keep scroll & `loadMore` is emitted even if api has not done. -> it leads few error
```suggestion
const isLoadMoreDone = ref(true)

const setDone = (value) => isLoadMoreDone.value = value;

const loadMore = (e) => {  
  if (!isLoadMoreDone.value) {
    return;
  }
  
  isLoadMoreDone.value = false;
  
  const scrollDifference = e.target?.scrollHeight - Math.round(e.target?.scrollTop)

  if (scrollDifference <= e.target?.clientHeight) {
    emit('loadMore', setDone)
  }
```

main app
```
<oc-select
  @scroll="handleScroll"
/>

const handleScroll = (done) => {
  api.get()
    .then((response) => {
       listData.value = response.data.items
       done(true);
    })
}
```